erna/scripts/upload.py

```diff
@@ -74,17 +74,27 @@ def xml(config, fact_tools_version, name, comment, xml_file):
 
 @click.command()
 @click.option('--config', help='Yaml file containing database credentials')
+@click.option('--version', help='Set the version, if not the jar is executed to obtain it')
 @click.argument('jar-file')
-def jar(config, jar_file):
+def jar(config, version, jar_file):
```

Comment:
While we are at it add a docstring that tells us what this function is for

Reply:
done